cylc/flow/config.py

```diff
@@ -1894,37 +1894,27 @@ def get_graph_raw(self, start_point_string, stop_point_string,
         self._last_graph_raw_edges = graph_raw_edges
         return graph_raw_edges
 
-    def get_graph_edges(self, start_point_string, stop_point_string):
+    def get_graph_edges(self, start_point, stop_point):
         """Convert the abstract graph edges (self.edges, etc) to actual edges
 
-        (This method differs from the get_graph_raw; class attributes are not
-        used to hold information from previous method calls.)
-
-        Actual edges have concrete ranges of cycle points.
-
+        This method differs from the get_graph_raw; class attributes are not
+        used to hold information from previous method calls, and only ungrouped
+        edges are returned.
         """
 
-        # Now define the concrete graph edges (pairs of nodes) for plotting.
-        if start_point_string in [None, '']:
-            return []
-        start_point = get_point(start_point_string)
-        actual_first_point = self.get_actual_first_point(start_point)
-
-        suite_final_point = get_point(
-            self.cfg['scheduling']['final cycle point'])
-
-        # Require a stop point determined by the data manager
-        if stop_point_string in [None, '']:
+        if not start_point:
             return []
-        stop_point = get_point(stop_point_string)
+        # Avoid infinite edge generation
         if not stop_point:
             return []
```

Comment:
Presumably this is only a development issue (i.e. correct use of the method), and you inherited the code, but shouldn't this raise an error rather than return no edges?

Reply:
Yes, should raise an error.. the points are provided via:
```
for point in self.pool_points.difference(old_pool_points):
            # All family & task cycle instances are generated and
            # populated with static data as 'ghost nodes'.
            self.generate_graph_elements(
                self.edges, self.task_proxies, self.family_proxies,
                point, point)
            has_updated = True
```
So could only break with development changes..